CorePreferences/src/au/gov/asd/tac/constellation/preferences/GraphPreferenceKeys.java

```diff
@@ -30,7 +30,7 @@ public final class GraphPreferenceKeys {
     public static final String BLAZE_OPACITY = "blazeOpacity";
     public static final int BLAZE_OPACITY_DEFAULT = 100;
     public static final String BLAZE_PRESET_COLORS = "blazePresetColors";
-    public static String BLAZE_PRESET_COLORS_DEFAULT = "#FF0000;#0000FF;#FFFF00;";
+    private static String BLAZE_PRESET_COLORS_DEFAULT = "#FF0000;#0000FF;#FFFF00;";

```

Comment:
similar case here (from what I can tell it seems the only reason it isn't final is for unit testing purposes)